server/environments.json

```diff
@@ -1,18 +1,18 @@
 {
   "development": {
     "authEndpoint": "https://sit-idp.nextcapital.com/as/token.oauth2",
-    "proxyEndpoint": "https://sit-pa.nextcapital.com"
+    "proxyEndpoint": "https://sit-app.nextcapital.com"
   },
   "sit": {
     "authEndpoint": "https://sit-idp.nextcapital.com/as/token.oauth2",
-    "proxyEndpoint": "https://sit-pa.nextcapital.com"
+    "proxyEndpoint": "https://sit-app.nextcapital.com"
   },
   "uat": {
     "authEndpoint": "https://uat-idp.nextcapital.com/as/token.oauth2",
-    "proxyEndpoint": "https://uat-pa.nextcapital.com"
+    "proxyEndpoint": "https://uat-app.nextcapital.com"
   },
   "production": {
     "authEndpoint": "https://prod-idp.nextcapital.com/as/token.oauth2",
-    "proxyEndpoint": "https://prod-pa.nextcapital.com"
+    "proxyEndpoint": "https://prod-app.nextcapital.com"
```

Comment:
This will only work with in the nextcapital network. If we want tit o work outside the nextcapital network we'll need to set up an API client in ping for this and then whitelist the IP addresses these requests are coming from.